isyfact-standards-doc/src/docs/antora/modules/changelog/pages/migrationsleitfaden.adoc

```diff
@@ -17,10 +17,292 @@ Die auf dieser Seite aufgeführten Hinweise sollen IsyFact Anwendungsentwicklern
 [[kapitel-batchrahmen]]
 === Batchrahmen
 
-
 [[kapitel-sicherheit]]
 === Sicherheit
```

Comment:
The subchapters of chapter `Bausteine` have a confusing order.
Suggestion in alphabetic order:
== Bausteine
=== Batchrahmen
=== Task Scheduler
=== Service
=== Sicherheit

Reply:
we could shift the reordering in a following ticket as soon as all chapter dependent changes were merged together. I think the reordering will be more easier in a completted document than in the current branch depending and splitted files.